unintended_ml_bias/bias_madlibs.py

```diff
@@ -144,8 +144,7 @@ def actual_label():
       if example not in example_set:
         example_set.add(example)
         template_count += 1
-        print(u'"{}",{},{}'.format(example, label,
-                                   template_pair['template']).encode('utf-8'))
+        print('"{}",{},{}'.format(example, label, template_pair['template']))
```

Comment:
WDYT `print(f'"{example}",{label},{template_pair['template']}'`? Might be clearer here. (LGTM regardless)

Reply:
I like the suggestion but you've got single quotes inside single quotes. Leaving cosmetic cleanups for later.